packages/kbn-server-route-repository/src/decode_request_params.ts

```diff
@@ -8,9 +8,9 @@
 import * as t from 'io-ts';
 import { omitBy, isPlainObject, isEmpty } from 'lodash';
 import { isLeft } from 'fp-ts/lib/Either';
-import { PathReporter } from 'io-ts/lib/PathReporter';
 import Boom from '@hapi/boom';
 import { strictKeysRt } from '@kbn/io-ts-utils';
+import { formatErrors } from '@kbn/securitysolution-io-ts-utils';
```

Comment:
don't we have a formatErrors function in `@kbn/io-ts-utils`? "We" (Observability) own that package. I would prefer that.

Reply:
Let me look into this

Reply:
Created an issue to do this as a follow up: https://github.com/elastic/kibana/issues/180808